### PR TITLE
Mention enum in metaclass example

### PR DESCRIPTION
**Pull request by mlouielu**

### CODE REVIEW

**Review by Mariatta — APPROVED** [review]

File: Doc/reference/datamodel.rst [hunk 1]
```diff
@@ -1886,8 +1886,8 @@ Metaclass example
 ^^^^^^^^^^^^^^^^^
 
 The potential uses for metaclasses are boundless. Some ideas that have been
-explored include logging, interface checking, automatic delegation, automatic
-property creation, proxies, frameworks, and automatic resource
+explored include enum, logging, interface checking, automatic delegation,
```
Comment (mlouielu) [inline Doc/reference/datamodel.rst:1889]: @Mariatta  Should this change to: `explored including: enum, logging....`

**Comment by Mariatta** [comment]
😱 I screwed up the commit message ...

**Comment by mlouielu** [comment]
@Mariatta maybe is time for a bot to check commit msg? :)  (could GitHub to such a hook?)

**Comment by Mariatta** [comment]
😅 I think it's the core dev's responsibility to write up sensible commit message.. So it's totally my fault, sorry.. 😔 At least I can fix the message in the backport branch...

**Pull request merged**